mono/utils/mono-threads-coop.h

```diff
@@ -53,14 +53,12 @@ mono_threads_safepoint (void)
 		mono_threads_state_poll ();
 }
 
-/* -1 and 0 also used:
- * -1 means uninitialized
- * 0 means unset
- */
 typedef enum {
+	MONO_THREADS_SUSPEND_UNINITIALIZED   = -1,
```

Comment:
Ideally uninitialized would be 0, but code depends on otherwise so leave it alone.